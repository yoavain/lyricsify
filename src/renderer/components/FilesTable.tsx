import Paper from "@material-ui/core/Paper";
import { makeStyles } from "@material-ui/core/styles";
import Table from "@material-ui/core/Table";
import TableBody from "@material-ui/core/TableBody";
import TableCell from "@material-ui/core/TableCell";
import TableHead from "@material-ui/core/TableHead";
import TableRow from "@material-ui/core/TableRow";
import * as moment from "moment";
import * as numeral from "numeral";
import * as React from "react";

interface Column {
    id: "filename" | "path" | "tag" | "title" | "artist" | "album" | "track" | "year" | "length" | "size" | "lastModified" | "hasLyrics";
    label: string;
    minWidth?: number;
    align?: "right";
    format?: (value: number) => string;
}

const columns: Column[] = [
    { id: "filename", label: "Filename", minWidth: 50 },
    { id: "path", label: "Path", minWidth: 50 },
    { id: "tag", label: "Tag", minWidth: 50 },
    { id: "title", label: "Title", minWidth: 50 },
    { id: "artist", label: "Artist", minWidth: 50 },
    { id: "album", label: "Album", minWidth: 50 },
    {
        id: "track",
        label: "Track #",
        minWidth: 50,
        align: "right",
        format: (value: number) => value.toLocaleString()
    },
    {
        id: "year",
        label: "Year",
        minWidth: 50,
        align: "right",
        format: (value: number) => value.toLocaleString()
    },
    {
        id: "length",
        label: "Length",
        minWidth: 50,
        align: "right",
        format: (value: number) => numeral(value).format("00:00:00")
    },
    {
        id: "size",
        label: "Size",
        minWidth: 50,
        align: "right",
        format: (value: number) => numeral(value).format("0.00b")
    },
    {
        id: "lastModified",
        label: "Last Modified",
        minWidth: 50,
        align: "right",
        format: (value: number) => moment(value).format("MM/DD/YYYY")
    },
    {
        id: "hasLyrics",
        label: "Has Lyrics",
        minWidth: 50,
        align: "right",
        format: (value: number) => moment(value).format("MM/DD/YYYY")
    }
];

interface RowData {
    filename: string;
    path: string;
    tag: string;
    title: string;
    artist: string;
    album: string;
    track: number;
    year: number;
    length: number;
    size: number;
    lastModified: number;
    hasLyrics: boolean;
}

function createData(
    filename: string,
    path: string,
    tag: string,
    title: string,
    artist: string,
    album: string,
    track: number,
    year: number,
    length: number,
    size: number,
    lastModified: number,
    hasLyrics: boolean = false
): RowData {
    return { filename, path, tag, title, artist, album, track, year, length, size, lastModified, hasLyrics };
}

const rows = [
    createData(
        "01 - Happens To The Heart.flac",
        "D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\",
        "FLAC",
        "Happens To The Heart",
        "Leonard Cohen",
        "Thanks For The Dance",
        1,
        2019,
        273,
        22749256,
        1574467200000
    ),
    createData("02 - Moving On.flac", "D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\", "FLAC", "Moving On", "Leonard Cohen", "Thanks For The Dance", 2, 2019, 192, 14556156, 1574467200000),
    createData(
        "03 - The Night of Santiago.flac",
        "D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\",
        "FLAC",
        "The Night of Santiago",
        "Leonard Cohen",
        "Thanks For The Dance",
        3,
        2019,
        255,
        20998200,
        1574467200000
    ),
    createData(
        "04 - Thanks For The Dance.flac",
        "D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\",
        "FLAC",
        "Thanks For The Dance",
        "Leonard Cohen",
        "Thanks For The Dance",
        4,
        2019,
        253,
        18920215,
        1574467200000
    ),
    createData("05 - It's Torn.flac", "D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\", "FLAC", "It's Torn", "Leonard Cohen", "Thanks For The Dance", 5, 2019, 178, 13998876, 1574467200000),
    createData("06 - The Goal.flac", "D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\", "FLAC", "The Goal", "Leonard Cohen", "Thanks For The Dance", 6, 2019, 72, 5576134, 1574467200000),
    createData("07 - Puppets.flac", "D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\", "FLAC", "Puppets", "Leonard Cohen", "Thanks For The Dance", 7, 2019, 160, 12369996, 1574467200000),
    createData("08 - The Hills.flac", "D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\", "FLAC", "The Hills", "Leonard Cohen", "Thanks For The Dance", 8, 2019, 258, 25574353, 1574467200000),
    createData(
        "09 - Listen To The Hummingbird.flac",
        "D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\",
        "FLAC",
        "Listen To The Hummingbird",
        "Leonard Cohen",
        "Thanks For The Dance",
        9,
        2019,
        121,
        7459732,
        1574467200000
    )
];

const useStyles = makeStyles({
    root: {
        width: "100%"
    },
    tableWrapper: {
        maxHeight: 440,
        overflow: "auto"
    }
});

export default function FilesTable() {
    const classes = useStyles();

    return (
        <Paper className={classes.root}>
            <div className={classes.tableWrapper}>
                <Table stickyHeader aria-label="sticky table">
                    <TableHead>
                        <TableRow>
                            {columns.map((column: Column) => (
                                <TableCell key={column.id} align={column.align} style={{ minWidth: column.minWidth }}>
                                    {column.label}
                                </TableCell>
                            ))}
                        </TableRow>
                    </TableHead>
                    <TableBody>
                        {rows.map((row: RowData) => {
                            return (
                                <TableRow hover role="checkbox" tabIndex={-1} key={row.filename}>
                                    {columns.map((column: Column) => {
                                        const value = row[column.id];
                                        return (
                                            <TableCell key={column.id} align={column.align}>
                                                {column.format && typeof value === "number" ? column.format(value) : value}
                                            </TableCell>
                                        );
                                    })}
                                </TableRow>
                            );
                        })}
                    </TableBody>
                </Table>
            </div>
        </Paper>
    );
}
