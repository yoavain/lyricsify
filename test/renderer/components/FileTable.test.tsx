import * as React from "react";
import * as renderer from "react-test-renderer";

import FilesTable from "../../../src/renderer/components/FilesTable";

describe("FilesTable component", () => {
    it("renders correctly", () => {
        const tree = renderer.create(<FilesTable />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
