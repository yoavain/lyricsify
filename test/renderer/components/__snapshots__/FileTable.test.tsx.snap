// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilesTable component renders correctly 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 makeStyles-root-1 MuiPaper-rounded"
>
  <div
    className="makeStyles-tableWrapper-2"
  >
    <table
      aria-label="sticky table"
      className="MuiTable-root MuiTable-stickyHeader"
    >
      <thead
        className="MuiTableHead-root"
      >
        <tr
          className="MuiTableRow-root MuiTableRow-head"
        >
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Filename
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Path
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Tag
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Title
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Artist
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Album
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Track #
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Year
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Length
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Size
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Last Modified
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader"
            scope="col"
            style={
              Object {
                "minWidth": 50,
              }
            }
          >
            Has Lyrics
          </th>
        </tr>
      </thead>
      <tbody
        className="MuiTableBody-root"
      >
        <tr
          className="MuiTableRow-root MuiTableRow-hover"
          role="checkbox"
          tabIndex={-1}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            01 - Happens To The Heart.flac
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            FLAC
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Happens To The Heart
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Leonard Cohen
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Thanks For The Dance
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            1
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2,019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0:04:33
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            22.75MB
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            11/23/2019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            No
          </td>
        </tr>
        <tr
          className="MuiTableRow-root MuiTableRow-hover"
          role="checkbox"
          tabIndex={-1}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            02 - Moving On.flac
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            FLAC
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Moving On
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Leonard Cohen
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Thanks For The Dance
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2,019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0:03:12
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            14.56MB
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            11/23/2019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            No
          </td>
        </tr>
        <tr
          className="MuiTableRow-root MuiTableRow-hover"
          role="checkbox"
          tabIndex={-1}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            03 - The Night of Santiago.flac
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            FLAC
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            The Night of Santiago
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Leonard Cohen
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Thanks For The Dance
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            3
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2,019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0:04:15
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            21.00MB
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            11/23/2019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            No
          </td>
        </tr>
        <tr
          className="MuiTableRow-root MuiTableRow-hover"
          role="checkbox"
          tabIndex={-1}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            04 - Thanks For The Dance.flac
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            FLAC
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Thanks For The Dance
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Leonard Cohen
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Thanks For The Dance
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            4
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2,019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0:04:13
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            18.92MB
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            11/23/2019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            No
          </td>
        </tr>
        <tr
          className="MuiTableRow-root MuiTableRow-hover"
          role="checkbox"
          tabIndex={-1}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            05 - It's Torn.flac
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            FLAC
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            It's Torn
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Leonard Cohen
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Thanks For The Dance
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            5
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2,019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0:02:58
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            14.00MB
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            11/23/2019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            No
          </td>
        </tr>
        <tr
          className="MuiTableRow-root MuiTableRow-hover"
          role="checkbox"
          tabIndex={-1}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            06 - The Goal.flac
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            FLAC
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            The Goal
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Leonard Cohen
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Thanks For The Dance
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            6
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2,019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0:01:12
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            5.58MB
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            11/23/2019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            No
          </td>
        </tr>
        <tr
          className="MuiTableRow-root MuiTableRow-hover"
          role="checkbox"
          tabIndex={-1}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            07 - Puppets.flac
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            FLAC
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Puppets
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Leonard Cohen
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Thanks For The Dance
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            7
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2,019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0:02:40
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            12.37MB
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            11/23/2019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            No
          </td>
        </tr>
        <tr
          className="MuiTableRow-root MuiTableRow-hover"
          role="checkbox"
          tabIndex={-1}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            08 - The Hills.flac
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            FLAC
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            The Hills
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Leonard Cohen
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Thanks For The Dance
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            8
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2,019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0:04:18
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            25.57MB
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            11/23/2019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            No
          </td>
        </tr>
        <tr
          className="MuiTableRow-root MuiTableRow-hover"
          role="checkbox"
          tabIndex={-1}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            09 - Listen To The Hummingbird.flac
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            D:\\Music\\Leonard Cohen\\2019 - Thanks For The Dance\\
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            FLAC
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Listen To The Hummingbird
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Leonard Cohen
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Thanks For The Dance
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            9
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2,019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0:02:01
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            7.46MB
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            11/23/2019
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            No
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
